packages/js/email-editor/src/components/personalization-tags/category-section.tsx

```diff
@@ -24,6 +26,16 @@ const CategorySection = ( {
 	closeCallback: () => void;
 	openLinkModal: ( tag: PersonalizationTag ) => void;
 } ) => {
+	const { updateBlockAttributes } = useDispatch( blockEditorStore );
+	const selectedBlockId = useSelect( ( select ) =>
+		select( blockEditorStore ).getSelectedBlockClientId()
+	);
+	const selectedBlock = useSelect( ( select ) =>
+		// @ts-expect-error getBlock expects one argument, but TS thinks it expects none
```

Comment:
Do you know why TS thinks that? Would it be possible to fix this globally, rather than adding a TS ignore?

Reply:
I came across this "fix" used elsewhere in the package and followed it, assuming the problem was in the original selector definition. On a second look after your comment, I found that we have type definitions for the `@wordpress ` modules in the email editor package, that are probably causing the conflict. I'll investigate further and try to fix it.

Reply:
I understand; I'm not a big fan of those comments. I'm okay with you keeping it here if you just add another instance of it, but we should probably make an effort to clean them up at some point.

Reply:
I did some digging and found that we override the module declaration for `@wordpress/data` and use `CurriedSelectorsOf` to transform the selector definitions. `CurriedSelectorsOf` is used to remove the first argument from the selector, which is expected to be `state`. However, in our case, we use the selectors types from `@types/wordpress__block-editor` which already have the state argument removed, and thus `CurriedSelectorsOf` removes another argument, which breaks the types.

Fixed it in https://github.com/woocommerce/woocommerce/pull/57958/commits/ed5d16536bd3da3cbb9b1667586d6307febeb8dc.